assert/assertions_test.go

```diff
@@ -555,7 +555,7 @@ func TestNoError(t *testing.T) {
 	}()
 
 	if err == nil { // err is not nil here!
-		t.Errorf("Error should be nil due to empty interface", err)
+		t.Errorf("Error should be nil due to empty interface: %s", err)
```

Comment:
nit: should use `%v` for types that aren't bytes or strings
same below at line 595